OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -196,6 +199,9 @@ public override Activity Tick(Actor self)
 					mobile.Facing,
 					0);
 
+				move.IsCanceled = this.IsCanceled;
```

Comment:
this.IsCanceled will always be false as we check if (IsCanceled) at the top of the method.